deps/mimalloc.version

```diff
@@ -0,0 +1,3 @@
+## jll artifact
+MIMALLOC_JLL_NAME := mimalloc
+
```

Comment:
```suggestion
# -*- makefile -*-
## jll artifact
MIMALLOC_JLL_NAME := mimalloc
```

Reply:
Yeah, this will need a bit of love, if we decide it's worth